packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldBasic.jsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import SearchFieldExampleTemplate from 'terra-search-field/lib/terra-dev-site/doc/example/SearchFieldExampleTemplate';
+import SearchField from 'terra-search-field/lib/SearchField';
 
-const SearchFieldExample = () => (
-  <SearchFieldExampleTemplate />
-);
+class SearchFieldBasic extends React.Component {
+  constructor(props) {
+    super(props);
+    this.invalidSearchMessage = 'The defualt minimum search length is 2.';
```

Comment:
It is a little strange to declare this string as a class property. A couple other options would be to declare a constant above or inline the message as a fallback.